src/main/resources/data-repository-openapi.yaml

```diff
@@ -659,6 +704,92 @@ definitions:
         items:
           $ref: '#/definitions/AssetModel'
 
+  ## Study Ingest Definitions ##
+  IngestRequestModel:
+    description: |
+      Request to ingest data from a file in GCS into a table in a study.
+      The ingest source must be readable by the DR manager.
+      TODO: other input forms
+    type: object
+    required:
+      - table
+      - path
+      - format
+    properties:
+      table:
+        $ref: '#/definitions/ObjectNameProperty'
+        description: name of the target table for ingest
+      path:
+        type: string
+        description: gs path to a file in a bucket accessible to data repo
+      format:
+        type: string
+        enum:
+          - csv
+          - json
+      load_tag:
+        type: string
+        description: |
+          client-specified tag for this load. If no id is specified, we use the
+          string form of the job create time as the tag.
+      max_bad_records:
+        type: integer
+        default: 0
+        description: max number of bad records to skip; applies to JSON and CSV
+      ignore_unknown_values:
+        type: boolean
+        default: true
+        description: skip extra data; applies to JSON and CSV
+      csv_field_delimiter:
+        type: string
+        default: ','
+        description: field separator
+      csv_quote:
+        type: string
+        default: '"'
+        description: quoting character
+      csv_skip_leading_rows:
+        type: integer
+        default: 0
+        description: number of header rows to skip
+      csv_allow_quoted_newlines:
+        type: boolean
+        default: true
+      csv_null_marker:
+        type: string
+        default: ''
+
+  IngestResponseModel:
+    description: |
+      Response from performing an ingest.
+    type: object
+    required:
+      - table
+      - path
+      - row_count
```

Comment:
i'm wondering why these are listed as required since this is a response. are the ones not listed as required sometimes not returned? if so, maybe we should add that into the description.

Reply:
Should we have any `required` on responses? I don't know if they have any effect.

Reply:
They will only be evaluated if we explicitly mark something handling the response with a `@Valid`  annotation -- I don't think there is any case where we are doing this now.